nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
I have not implemented parallelism with the queue for the `'classic'` proposal, so right now it runs in serial as normal.